go-controller/pkg/ovn/ovn.go

```diff
@@ -494,3 +429,73 @@ func newEgressServiceController(client clientset.Interface, nbClient libovsdbcli
 		svcFactory.Discovery().V1().EndpointSlices(),
 		svcFactory.Core().V1().Nodes())
 }
+
+func (oc *DefaultNetworkController) syncNetworkPolicies(networkPolicies []interface{}) error {
```

Comment:
I am not sure what is our common strategy, but I think it may be better to leave these netpol-specific function under base_network_controller_policy, otherwise all the default controller specific functions will flood this file, and then it is difficult to find them
wdyt @cathy-zhou @jcaamano ?

Reply:
probably to default_network_controller.go? this is not for base_network_controller, so it should not belong to base_network_controller_policy.go

Reply:
if we had proper packaging, every file would be split into bnc/<feature_name>, default_network_controller/<feature_name> and where needed same for the secondary controllers. Now when we put the default network controller parts to the same file (ovn or default_network_controller) it may difficult to split them back when/if we change the packages structure.
I agree that it doesn't make a lot of sense to put these function under bnc_ files, so we can make default_network_controller_policy file for that purpose, wdyt?

Reply:
create a file only for one function? I am not sure.

Reply:
I guess it should be based on if we are planning to have the same separate file in the future under default network controller package, or if we want to leave all functions in one big file . For now e.g. I am not sure what is the difference between `master.go` `ovn.go` and `default_network_controller.go`, so what is the right place for default network controller netpol functions (that also includes `addHairpinAllowACL`)
@jcaamano any thoughts?

Reply:
We can make default_network_controller_policy with WatchNetwoerkPolicies, addHairpinAllowACL and syncNetworkPolicies.